tests/test_backend.py

```diff
@@ -607,6 +607,42 @@ def func(x, output_shape_placeholder):
         self._run_test_case(func, [_OUTPUT], {_INPUT: x_val, _INPUT1: output_shape},
                             rtol=1e-05, process_args=process_args)
 
+    def test_conv2d_quantization_axis(self):
```

Comment:
Please add an annotation for this function:

@check_opset_min_version(10, "quantize_and_dequantize")